### ISSUE TITLE
Pop over consistencies

### ISSUE THREAD

**Issue by sreenug** [digitalgreenorg/dg#72]
There seems to have been some confusion on pop-up comment from Rikin: We should still have icons for attendance, interests, questions, and adoptions on the videos thumbnails but there should be on-hover pop-ups (as opposed to ALT TEXT).  This should be uniform across the partners, villages, groups, andfarmer pages.  (The current version shows icons with ALT TEXT at the farmer-level and pop-ups and no icons at the village, partner, etc.-level)
